backend/middleware/errorHandler.js

```diff
@@ -1,18 +1,21 @@
-const { logError } = require('../logError')
+const { isXHRRequest } = require('../utils')
 
-module.exports = (error, req, res, next) => {
-  if (error?.response?.status === 404) {
-    res.status(404)
-    return res.render('notFound.njk', { url: req.headers.referer || '/' })
-  }
+module.exports = ({ logError }) => (error, req, res, next) => {
+  const status = error?.response?.status || 500
+
+  if (status >= 500) logError(req.originalUrl, error, 'There was a problem loading page')
 
-  const pageData = {
-    url: res.locals.redirectUrl || req.originalUrl,
-    homeUrl: res.locals.homeUrl,
+  if (isXHRRequest(req)) {
```

Comment:
Without this the server can't deal with errors caused by ajax calls, thus causing the app/server to crash.